src/components/views/dialogs/ChatInviteDialog.js

```diff
@@ -135,12 +135,26 @@ module.exports = React.createClass({
         } else if (e.keyCode === 13) { // enter
             e.stopPropagation();
             e.preventDefault();
-            this.onButtonClick();
```

Comment:
so previously, we submitted the form if the input box was empty. I think that is desirable behaviour. Can we keep it?